### PR TITLE
Add an amoled black theme

### LINKED ISSUE #155: Add the amoled theme
It's a simple suggerition

### PR DESCRIPTION
**Pull request by Bnyro**
This PR adds a switch to enable a pure black (amoled) theme, works for both the Obtainium and the Material You theme.

closes #155

### CODE REVIEW

File: assets/translations/en.json [hunk 1]
```diff
@@ -122,6 +122,7 @@
     "followSystem": "Follow System",
     "obtainium": "Obtainium",
     "materialYou": "Material You",
+    "amoledTheme": "Pure black theme",
```
Comment (ImranR98) [inline assets/translations/en.json:125]: Could you change this to "Use pure black dark theme" - this makes it more obvious that the toggle will only have an effect when the dark theme is on.

The key (`amoledTheme`) should also be closer to the value. Maybe `useBlackTheme`?

Also please add to all other language files (re-use the English version as a placeholder for others to update later).

File: lib/pages/settings.dart [hunk 2]
```diff
@@ -226,6 +226,18 @@ class _SettingsPageState extends State<SettingsPage> {
                             height16,
                             colourDropdown,
                             height16,
+                            Row(
```
Comment (ImranR98) [inline lib/pages/settings.dart:229]: This should be directly below the `themeDropdown` as the two settings are related.

File: lib/pages/settings.dart [hunk 3]
```diff
@@ -226,6 +226,18 @@ class _SettingsPageState extends State<SettingsPage> {
...
+                              children: [
+                                Text(tr('amoledTheme')),
+                                Switch(
+                                    value: settingsProvider.amoledDarkTheme,
+                                    onChanged: (value) {
+                                      settingsProvider.amoledDarkTheme = value;
+                                    })
+                              ],
+                            ),
+                            height16,
```
Comment (ImranR98) [inline lib/pages/settings.dart:240]: `height16` won't be necessary anymore as the `colourDropdown` has enough space above it.

**Review by ImranR98 — CHANGES_REQUESTED** [review]
Thanks for the PR! Just a few comments to review.

**Comment by Bnyro** [comment]
Thanks for the review, I force pushed the suggested changes!

**Review by ImranR98 — APPROVED** [review]

**Pull request merged**